pnl-subgraph/schema.graphql

```diff
@@ -0,0 +1,29 @@
+# User Token Position
+type UserPosition @entity {
```

Comment:
Might want the tokenid explicitly here as a field, so we can easily filter for specific markets